js/setup.js

```diff
@@ -62,7 +68,19 @@ const mapStateToProps = state => ({
   nav: state.nav
 })
 
-const AppWithNavigationState = connect(mapStateToProps)(AppNavigation)
+// This add a connectivity listener
+// Using the parameter `withRedux: true` we save the network state into the store
+const AppWithConnectivity = withNetworkConnectivity({
+  withRedux: true,
+  timeout: 5000,
```

Comment:
would be awesome to have these parameters somewhere in a config file, or at least defined as constants on top of this file